src/test/java/racingcar/car/CarNamesTest.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class CarNamesTest {
+    @DisplayName("이름문자열을 받아서 CarName List를 가진 1급 컬랙션 생성")
+    @Test
+    void newCarNamesWithStringValue() {
+        //given
+        String nameValue = "1,2,3,4";
+        int expectedSize = 4;
+
+        //when
+        CarNames carNames = CarNames.newInstance(nameValue);
+
+        //then
+        assertThat(carNames).isNotNull();
+        assertThat(carNames.getNames().size()).isEqualTo(expectedSize);
```

Comment:
hasSize 를 이용할 수도 있어요!
```suggestion
        assertThat(carNames.getNames()).hasSize(expectedSize);
```

Reply:
오,,, 감사합니다!